contracts/v2/PolygonRollupManager.sol

```diff
@@ -231,6 +231,8 @@ contract PolygonRollupManager is
     bytes32 internal constant _EMERGENCY_COUNCIL_ADMIN =
         keccak256("EMERGENCY_COUNCIL_ADMIN");
 
+    string internal constant _ROLLUP_MANAGER_VERSION_PESSIMISTIC = "pessimistic";
```

Comment:
maybe it's worth it to have a constnat that always have the same name, and public, to be able to easy retrieval from the node :thinking: something like currentVersion!?¿